### ISSUE TITLE
No ads shown on Linux-Ubuntu 

### ISSUE THREAD

**Issue by furkanguzel161** [brave/brave-browser#25920]
## Description 

I am getting no ads on my desktop pc but I installed brave on my phone I got an ad without a problem.
I have been using it for a few hours now.
I checked the FAQ but its mostly for windows and macOS also I have a gaming mouse g402 and I have piper installed
https://community.brave.com/t/no-ads-shown-on-linux-ubuntu/

## Steps to Reproduce

   1. just use the browser and you will get no ads

## Actual result:

No ads

## Expected result:
Ads should be shown on the notification of ubuntu

## Brave version (brave://version info)
 
Ubuntu 22.04.1 LTS x86_64 - Brave Version 1.44.108

## Version/Channel Information:

- Can you reproduce this issue with the current release?  yes
- Can you reproduce this issue with the beta channel?  Did not test it, I fear my browser history bookmark, etc will be lost.
- Can you reproduce this issue with the nightly channel? Did not test it.

**Comment by furkanguzel161** [comment]
Update!
switching to X11 solves this issue it seems there is a problem with Wayland...

**Comment by TMTRTLP** [comment]
Yeah, I also appear to be able to partially reproduce this on the current release (1.45.116) when using Wayland on Ubuntu 22.04.1 and 22.10 (I upgraded to 22.10 this week).  I don't get push notification ads while actually interacting with the browser and do end up getting them after I wake my computer up from a power saving mode when the browser was left open.  It's almost like Brave gets confused whether you're interacting with it or not when on Wayland and displays those ads at the wrong time.  All other ads on the home screen do seem to be working perfectly fine though.

Like the above comment switching to X11 causes the ads to behave in the expected way.
